client/src/components/Auth/LoginForm.js

```diff
@@ -0,0 +1,338 @@
+      return false;
+    }
+
+    return true;
+  };
+
+  const onChange = (stateName) => (event) => {
+    setState({ ...state, [stateName]: event.target.value });
+  };
+
+  const onSubmit = (event) => {
+    event.preventDefault();
+    if (validate()) {
+      // TODO: send request onSubmit
+      props.login(state);
+      props.closeModal();
+    }
+  };
+
+  return (
+    <Grid
+      component="form"
+      container
+      direction="row"
+      justify="center"
+      alignItems="flex-start"
+      className={classes.container}
+      action="#"
+    >
+      <Grid
+        container
+        className={classes.container__body}
+        direction={matchMobile ? 'column' : 'row'}
+        justify="center"
+        alignItems="center"
+      >
+        <Grid
+          container
+          direction="column"
+          justify="flex-start"
+          alignItems="center"
+          lg={5}
+          md={5}
+          xs={12}
+        >
+          <Box className={classNames(classesMobile.title)}>
+            <Typography className={classes.title} style={tungsten} variant="h3" color="textPrimary" align="center">
+              Log In
+            </Typography>
+            {
+              matchMobile ? (
+                <Typography
+                  onClick={() => props.registerModal()}
+                  className={classNames(classes.title, classesMobile.inactive)}
+                  style={tungsten}
+                  variant="h3"
+                  color="textPrimary"
+                  align="center"
+                >
+                  Create Account
+                </Typography>
+              ) : null
+            }
+          </Box>
+          <FormControl className={classes.formControl} variant="outlined">
+            <InputLabel classes={{ root: classes.label }} htmlFor="loginOrEmail">
+              Login or Email
+            </InputLabel>
+            <OutlinedInput
+              id="loginOrEmail"
+              classes={{
+                root: classes.inputRoot,
+                input: classes.input,
+              }}
+              value={state.loginOrEmail}
+              onChange={onChange('loginOrEmail')}
+              labelWidth={105}
+              fullWidth
+            />
+          </FormControl>
+          <FormControl className={classes.formControl} variant="outlined">
+            <InputLabel classes={{ root: classes.label }} htmlFor="password">
+              Password
+            </InputLabel>
+            <OutlinedInput
+              id="password"
+              type={passwordHidden ? 'password' : 'text'}
+              classes={{
+                root: classes.inputRoot,
+                input: classes.input,
+              }}
+              value={state.password}
+              onChange={onChange('password')}
+              labelWidth={105}
+              fullWidth
+              endAdornment={
+                state.password
+                  ? (
+                    <PasswordIcon
```

Comment:
password icon has to be with the property (cursor: pointer;).
visibility of password is broken with resolution lower than 900px.

Reply:
Resolved